src/ampPage.js

```diff
@@ -12,6 +12,8 @@ export default (body, style, options = {}) =>
 <html amp lang="en">
   <head>
     <meta charset="utf-8">
+    ${options.useAmpClientIdApi ?
```

Comment:
should we maybe instead allow for a generic mechanism to supply N custom meta name/content values? like `metas: [{ name: 'google', content: 'analytics' }]` or something?